modules/virt-editing-vm-web.adoc

```diff
@@ -14,31 +14,37 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machines
-:object-vm-overview: VM Template Details
+:object-gui: Templates
+:object-vm-details: VM Template Details
 endif::[]
 
 ifeval::["{context}" == "virt-edit-vms"]
 :virt-vm:
 :object: virtual machine
 :object-gui: Virtual Machines
-:object-vm-overview: Virtual Machine Overview
+:object-vm-details: Details
 endif::[]
 
 [id="virt-editing-vm-web_{context}"]
 
 = Editing a {object} in the web console
 
-Edit select values of a {object} that displays in the *{object-vm-overview}* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.
+Edit select values of a {object} in the web console by clicking the pencil icon next to the relevant field. Other values can be edited using the CLI.
 
 Labels and annotations are editable for both preconfigured Red Hat templates and your custom virtual machine templates. All other values are editable only for custom virtual machine templates that users have created using the Red Hat templates or the *Create Virtual Machine Template* wizard.
 
 .Procedure
 
 . Click *Workloads* -> *Virtualization* from the side menu.
+
 . Click the *{object-gui}* tab.
-. Select a {object} to open the *Virtual Machine Overview* screen.
+
+. Select a {object}.
+
+. Click the *object-vm-details* tab.
```

Comment:
This needs to be wrapped in {} so it's recognise as a variable

Reply:
Good catch!

Reply:
Super catch! I am on it.